translator/source_code/icone_translator.py

```diff
@@ -37,11 +38,25 @@ def wzdx_creator(messages, info):
     if info['metadata'].get('datafeed_frequency_update', False):
         data_source['update_frequency'] = info['metadata']['datafeed_frequency_update']
     data_source['update_date'] = datetime.now().strftime("%Y-%m-%dT%H:%M:%SZ")
-    data_source['location_verify_method'] = info['metadata']['location_verify_method']
+    # data_source['location_verify_method'] = info['metadata']['location_verify_method']
```

Comment:
There's a large amount of commented code in here that doesn't seem to be needed. Can we clean that up and remove?